dom.go

```diff
@@ -2851,3 +2850,35 @@ func (css *CSSStyleDeclaration) Length() int {
 type Text struct {
 	*BasicNode
 }
+
+type history struct {
+	*js.Object
+}
+
+func (h *history) Length() int {
+	return h.Get("state").Int()
+}
+
+func (h *history) State() interface{} {
+	return h.Get("state")
+}
+
+func (h *history) Back() {
+	h.Object.Call("forward")
```

Comment:
Style suggestion. For consistency with above `h.Get` and the rest of the package, you can skip skip `.Object` since it's an embedded struct, just do `h.Call("forward")`.

Here, and in 4 places below.